examples/testing/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_library", "ts_test")
+
+ts_library(
+    name = "lib",
+    srcs = ["decrement.ts"],
+)
+
+ts_library(
+    name = "tests",
+    srcs = ["decrement.spec.ts"],
+    deps = [":lib"],
+    testonly = 1,
+)
+
+ts_test(
```

Comment:
This doesn't run tests by itself right?
Which rule consumes this?

Reply:
Yes, this does run the tests. I'll add a README about it